requirements.txt

```diff
@@ -4,3 +4,6 @@ https://github.com/cta-sst-1m/protozfitsreader/archive/v0.44.1.tar.gz
 -e git+https://github.com/cta-observatory/ctapipe-extra.git#egg=ctapipe-extra
 -e git+https://github.com/cta-sst-1m/CTS.git#egg=CTS
 -e git+https://github.com/dstndstn/astrometry.net.git#egg=astrometry
+https://github.com/calispac/histogram/archive/0.1.1.tar.gz
+-e git+https://github.com/iminuit/probfit.git#egg=probfit
+-e git+https://github.com/uvemas/ViTables.git#egg=vitables
```

Comment:
vitables seems not to be used anywhere .. do we need it?

Reply:
yes ! do you want to remove it from requirements ?

Reply:
It is a bit crazy to put recommendations into requirements....